pkg/scheduler/framework/plugins/noderesources/least_allocated_test.go

```diff
@@ -90,10 +91,15 @@ func TestNodeResourcesLeastAllocated(t *testing.T) {
 			},
 		},
 	}
+	defaultResourceLeastAllocatedSet := []config.ResourceSpec{
+		{Name: "cpu", Weight: 1},
+		{Name: "memory", Weight: 1},
```

Comment:
use v1.ResourceCPU and v1.ResourceMemory instead of "cpu" and "memory"?

Reply:
done